query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -20,6 +22,27 @@ func init() {
 	)
 }
 
+type FromStorageRule struct{}
+
+func (rule FromStorageRule) Name() string {
+	return "influxdata/influxdb.FromStorageRule"
+}
+
+func (rule FromStorageRule) Pattern() plan.Pattern {
+	return plan.Pat(influxdb.FromKind)
+}
+
+func (rule FromStorageRule) Rewrite(node plan.Node) (plan.Node, bool, error) {
+	fromSpec := node.ProcedureSpec().(*influxdb.FromProcedureSpec)
```

Comment:
Just to be clear, this takes the `from` in the `flux` repo, which is the remote-host `from`, and converts it to an `influxdb` from, which will talk to storage directly.

Reply:
This is correct.